go.mod

```diff
@@ -214,6 +214,3 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 // We are using a fork of the migration library with custom functionality that
 // did not yet make it into the upstream repository.
 replace github.com/golang-migrate/migrate/v4 => github.com/lightninglabs/migrate/v4 v4.18.2-9023d66a-fork-pr-2
-
-// Note this is a temproary replace and will be removed when taprpc is tagged.
-replace github.com/lightninglabs/taproot-assets/taprpc => ./taprpc
```

Comment:
This will make future PRs that change anything in the `taprpc` package super hard (needs to be two PRs). So I think we decided to leave this in. But we should update the version on line 32 regularly so external consumers will pull in the correct version.